content/docs/reference/version-policy.md

```diff
@@ -0,0 +1,229 @@
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kubeflow/tree/master/components/profile-controller">Profile 
+        Controller</a> (<a href="/docs/other-guides/multi-user-overview/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/pytorch-operator">PyTorch operator</a> (<a href="/docs/components/training/pytorch/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td>PyTorch serving 
+          (<a href="/docs/components/serving/pytorchserving/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/batch-predict">TensorFlow 
+          batch prediction</a>
+          (<a href="/docs/components/serving/tfbatchpredict/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td>TensorFlow serving (<a href="/docs/components/serving/tfserving_new/">docs</a>)</td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/tf-operator">TFJob operator</a>
+          (<a href="/docs/components/training/tftraining/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/xgboost-operator">XGBoost operator</a></td>
+        <td>Experimental</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<a id="sdk-matrix"></a>
+## Kubeflow SDKs and CLIs
+
+Alongside Kubeflow {{% kf-latest-version %}}, you may want to use 
+one of the following Kubeflow SDKs and command-line interfaces 
+(CLIs).
+
+<div class="table-responsive">
+  <table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>SDK / CLI</th>
+        <th>Status in Kubeflow {{% kf-latest-version %}}</th>
+        <th>SDK/CLI version</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><a href="https://github.com/kubeflow/fairing">Fairing</a> (<a href="/docs/fairing/reference/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+      <tr>
+        <td><a href="https://github.com/kubeflow/kfctl">kfctl</a> (<a href="/docs/other-guides/kustomize/">docs</a>)</td>
+        <td>Stable</td>
+        <td>1.0</td>
+      </tr>
+      <tr>
+        <td><a href="https://kubeflow-pipelines.readthedocs.io/en/latest/">Kubeflow Pipelines SDK</a> (<a href="/docs/pipelines/sdk/">docs</a>)</td>
+        <td>Beta</td>
+        <td></td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Support levels
+
+The expectations for supportability and the types of support available depend
+on the stable status of each application or other component.
+For more information, see the support details page.
```

Comment:
Can you link to the support details page?

Reply:
Ha :) We don't have it yet. https://github.com/kubeflow/website/issues/1131

Reply:
The page now includes the link to the support details page. Note, though, that the link won't work as the support page is created by a different PR. Here's a preview:
https://deploy-preview-1520--competent-brattain-de2d6d.netlify.com/docs/other-guides/support/